projects/cdk/abstract/dialog.service.ts

```diff
@@ -37,6 +37,7 @@ export abstract class AbstractTuiDialogService<T extends {}> extends Observable<
                 completeWith,
                 $implicit: observer,
                 component: this.component,
+                createdAt: Date.now(),
```

Comment:
Maybe we can just use id below? It's autogenerated based on ever increasing increment and `Date.now()`

Reply:
i'm not sure that it is good idea.

If I use id I will need to remove `tui_` prefix from id (before converting to the number).

Moreover, does it violate the principles of Dependency Injection if component knows how  TuiIdService generate id?
TuiIdService can be changed in future that will break logic inside dialogs.